cityofaustin/atd-moped
# adds public process status to view, filter, csv

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/10834

this pr adds the field 'public process status' to the project list view (hidden by default), adds an advanced filter with autocomplete, and adds it to the csv export

## Testing
**URL to test:** 

[10834_public_process_status_search_view--atd-moped-main.netlify.app/moped/](https://10834_public_process_status_search_view--atd-moped-main.netlify.app/moped/)

**Steps to test:**
- on the project list view table, enable the new 'public process status' column
- scroll to the right and note the values. if there is no public process status, you should see '-'
- go to a project or multiple and add public process statuses
- go back to the project list view and note the changes
- export the table to csv and note the new column and values
- click the add filter button and select field 'public process status'
- try all the operators with input that should yield results
- selecting the 'is' and 'is not' operators should render dropdowns of all available public process statuses

---
#### Ship list
